M2/Macaulay2/d/actors3.d

```diff
@@ -2346,6 +2363,20 @@ scan(e:Expr):Expr := (
      else WrongNumArgs(2));
 setupfun("scan",scan);
 
+-- # typical value: scanPairs, Thing, Function, Nothing
```

Comment:
Since this line is for readability of documentation and doesn't have to be literally true, I wonder if it would be useful to use a dummy `Iterable` class instead of `Thing` for things like this. I know that practically anything with an `iterator` method defined on it can be iterable, so it may cause some confusion.